opm/grid/GraphOfGridWrappers.cpp

```diff
@@ -591,7 +591,7 @@ zoltanPartitioningWithGraphOfGrid(const Dune::CpGrid& grid,
 
     // prepare graph and contract well cells
     // non-root processes have empty grid and no wells
-    GraphOfGrid gog(grid);
+    GraphOfGrid gog(grid,transmissibilities);
```

Comment:
Nitpick: space after the comma please. Also elsewhere. This will will make it a lot easier to read.